bundles/org.openhab.binding.openwebnet/pom.xml

```
@@ -23,7 +23,7 @@
     <dependency>
       <groupId>com.github.openwebnet4j</groupId>
```

```suggestion
      <groupId>openwebnet4j</groupId>
```
This should make it resolve from our Artifactory for now.